docs/blog/newsletter-september-2017.rst

```diff
@@ -0,0 +1,83 @@
+#####################################
+Write the Docs Newsletter – September
+#####################################
+
+Hi everybody!
+
+Kelly O'Brien here, your friendly neighborhood newsletter editor, wishing you all a very happy September! As it happens, September is Be Kind to Editors & Writers Month (h/t @monique.semp for the discovery)! So make sure you take a moment to be kind to yourself and to the documentarians around you!
+
+Another exciting fact about September: It's the month that Write the Docs Prague happens! In fact, the conference is very nearly upon us! Folks have started to make their way to Prague (I'm hopping on a train this afternoon!), and the conference chatter is starting to pick up in the #wtd-conferences channel on Slack, if you're of a mind to tune in.
```

Comment:
lower case i in "it's" here?

Reply:
Also the whole clause feels a bit weak. Maybe instead something like 
"It's the month of [| for] Write the Docs Prague!"
"Write the Docs Prague takes place this month!"
"It's the month that Write the Docs Prague takes place!"

Reply:
Ah, good call, definitely needs to be punchier.